logstash-core/src/main/java/org/logstash/Main.java

```diff
@@ -0,0 +1,53 @@
+package org.logstash;
+
+import java.io.InputStream;
+import org.jruby.Ruby;
+import org.jruby.RubyException;
+import org.jruby.RubyInstanceConfig;
+import org.jruby.RubyNumeric;
+import org.jruby.exceptions.RaiseException;
+import org.jruby.runtime.builtin.IRubyObject;
+
+public final class Main implements Runnable {
```

Comment:
Let's call this `Logstash` so that when folks do `jps` "Logstash" is what shows up instead of "Main"